nncf/torch/nncf_network.py

```diff
@@ -338,10 +338,36 @@ def insert_at_point(self, point: PTInsertionPoint, fn_list: List[Callable]):
             raise RuntimeError("Unsupported insertion type: {}".format(point.insertion_type))
 
     def __getattr__(self, name):
-        wrapped_module = super().__getattr__(MODEL_WRAPPED_BY_NNCF_ATTR_NAME)
-        if hasattr(wrapped_module, name):
-            return getattr(wrapped_module, name)
-        return super().__getattr__(name)
+        class NotFound:
```

Comment:
Why we can not use `None`?

```python
def get_nncf_network_attr(self, name):
    return self.__dict__.get(name)


attr = get_nncf_network_attr(self, name)

if attr is not None:
    return attr
```

Reply:
If we would return None we can't distinguish cases when the attribute exists and its value is None and there is no such attribute.